python/tvm/micro/model_library_format.py

```diff
@@ -126,20 +125,25 @@ def export_model_library_format(mod: graph_executor_factory.GraphExecutorFactory
 
     Parameters
     ----------
-    mod : tvm.relay.backend.graph_executor_factory.GraphExecutorFactoryModule
+    mod : tvm.relay.backend.executor_factory.ExecutorFactoryModule
         The return value of tvm.relay.build, which will be exported into Model Library Format.
     file_name : str
         Path to the .tar archive to generate.
     """
     tempdir = utils.tempdir()
+    is_aot = isinstance(mod, executor_factory.AOTExecutorFactoryModule)
```

Comment:
Can you help me understand here? I need the `is_aot` logic, to avoid generating the memory path and to select the correct runtime. Either I add a `get_factory_type` from the factory or I inspect its type here

Reply:
oh i'm sorry, I misread. ignore :)